README.md

```diff
@@ -1,45 +1,83 @@
 # MVP III
 
+## Background
+
+Code and instructions for building the 'brain' of the controled environment hydroponics unit.
+It is mostly a collection of python code that runs on a Raspberry Pi (or similar device). See the OpenAg [forums](http://forum.openag.media.mit.edu/) for discussion and issues:
+
+The MVP (Minimal Viable Product) is a simplified version of the MIT OpenAg Food Computer.
+
+## Installation
+
+- ### Set up the Raspberry Pi with the NOOBS installation
+
+  - You can use the instructions in the official Rasbperry Pi [quickstart guide](https://www.raspberrypi.org/learning/software-guide/quickstart/) under 'Install Raspbian with NOOBS'
+  - Follow the instructions in the official documentation to download the NOOBS zip file, unzip the file, and copy the files inside the unzipped folder to the SD card.
+
+- ### Set up the operating system on the Rasbperry Pi.
+
+  - Once the SD card is set up with the NOOBS installation, insert it in the Raspberry Pi. Connect the Rasbperry Pi to a mouse, keyboard, and monitor. Power up the Raspberry Pi.
+
+  - Follow the onscreen prompts, connecting to wifi if not on a wired connection.
+
+  - Be sure to set the keyboard to the correct locale.
+
+  - Continue to follow the onscreen prompts using the recommended raspbian installation.
+
+- ### Prepare Raspberry pi
+
+  - Open the terminal application and run `sudo raspi-config`
+  - Under `4-Localization`, change the timezone to your country and area. To navigate through menus use your keyboard's arrow keys.
+  - Back in the main config menu, under `5-Interface` enable Camera, SSH, I2C, and 1 wire.
+  - In the config menu, hit `Finish` at the bottom right to exit, and reboot when prompted.
+
+- ### Install MVP software
+  - In the terminal, run the following to download and run the bash script:
+  ```
+  bash <(curl -s https://raw.githubusercontent.com/futureag/mvp/master/install.sh)
+  ```
+
+## Release info
+
 Latest Release: 3.1.6
 
 Change Log: 2018/10/2
- - Added binary CouchDB to get around build problems
- - NOTE: Fuxton is not included in this binary
+
+- Added binary CouchDB to get around build problems
+- NOTE: Fuxton is not included in this binary
 
 Change Log: 2018/09/24
- - add chmod to Startup.sh so logs always have the correct write permission
+
+- add chmod to Startup.sh so logs always have the correct write permission
 
 Change Log: 2018/09/21
-  - Correct typo in Startup.sh
+
```

Comment:
I think my editor added some auto-formatting, its just some spacing and removal of extra whitespace